content/docs/concepts/data_model.md

```diff
@@ -37,6 +37,24 @@ are reserved for internal use.
 
 Label values may contain any Unicode characters.
 
+A label with an empty label value is considered equivalent to a label that does not exist. For example:
```

Comment:
We don't have extended examples on this page, and this example features PromQL which isn't part of our data model. Just this sentence should do.

Reply:
ok